service/src/main/kotlin/fi/espoo/evaka/serviceneednew/NewServiceNeedController.kt

```diff
@@ -89,6 +93,7 @@ class NewServiceNeedController(
                 shiftCare = body.shiftCare,
                 confirmedAt = HelsinkiDateTime.now()
             )
+            tx.getNewServiceNeedChildRange(id).let { notifyServiceNeedUpdated(tx, asyncJobRunner, it) }
```

Comment:
We need to use the combined date range of the old and the new versions' dates to cover all the dates where service need was changed.

Reply:
done